regenerate.sh

```diff
@@ -27,8 +29,16 @@ tv_scripts=(
     zip_0244
     zip_0316)
 
+formats="${*:-rust json zcash}"
```

Comment:
I had a bug here which I've force-pushed to correct: `${1:-rust json zcash}` -> `${*:-rust json zcash}`.